clients/cobol-lsp-vscode-extension/src/task/CustomBuildTaskTerminal.ts

```diff
@@ -0,0 +1,142 @@
+        let job = await jesApi.getJob(submitJob.jobid);
+        do {
+            job = await jesApi.getJob(submitJob.jobid);
+        } while (!job.retcode);
+
+        this.writeEmitter.fire(`JOB completed, return code: ${job.retcode} ...\r\n`);
+        if(`${job.retcode}` !== "JCL ERROR") {
+        const sysPrintSpool = (await jesApi.getSpoolFiles(submitJob.jobname, submitJob.jobid))
+            .filter(file => file.ddname === "SYSPRINT");
+        const sysPrintContent = await jesApi.getSpoolContentById(submitJob.jobname, submitJob.jobid, sysPrintSpool[0].id);
+
+        this.writeEmitter.fire("Analysing output...\r\n");
+
+        this.writeEmitter.fire(sysPrintContent.split("\n")
+            .filter(str => str.match(this.ERROR_REGEX))
+            .map(str => {
+                const match = str.match(this.ERROR_REGEX);
+                return `\x1b[4m\x1b[33m${this.documentText.uri.fsPath}:${match[1]}:7\x1b[24m\x1b[39m -72 - ${match[2]}: ${match[3]}`;
+            })
+            .join("\r\n"));
+            this.storeSysPrint(submitJob, sysPrintContent);
+        }
+    }
+
+    private storeSysPrint(submitJob: any, sysPrintContent: any) {
+        if (this.definition.saveSysprint) {
+            const fileLocation = this.definition.sysPrintLocation ? `${this.definition.sysPrintLocation}` : `${vscode.workspace.workspaceFolders[0].uri.fsPath}${path.sep}SYSPRINT`;
+            if (!fs.existsSync(fileLocation)) {
+                fs.mkdirSync(fileLocation, { recursive: true });
+            }
+            fs.writeFileSync(`${fileLocation}${path.sep}${submitJob.jobid}`, sysPrintContent);
+            this.writeEmitter.fire("\r\n");
+            this.writeEmitter.fire(`SYSPRINT at \x1b[4m\x1b[33m${fileLocation}${path.sep}${submitJob.jobid}\x1b[24m\x1b[39m\r\n`);
+        }
+    }
+
+    private getCompileJobJCL() {
+        let jclToSubmit: string;
+        const jobCard = fs.readFileSync(path.resolve(vscode.workspace.workspaceFolders[0].uri.fsPath, this.definition.jobCard)).toString().trim();
+        const steplibs = this.definition.steplib && this.definition.steplib.length > 0 ? this.generateLibs("STEPLIB", this.definition.steplib) : "//*";
+        const syslibs = this.definition.syslib && this.definition.syslib.length > 0 ? this.generateLibs("SYSLIB", this.definition.syslib) : "//*";
+        jclToSubmit = `${jobCard}\r\n${COMPILE_JCL_HEADER}\r\n${steplibs}\r\n${COMPILE_JCL_MIDDLE}\r\n${syslibs}\r\n${COMPILE_JCL_TAIL}\r\n${this.documentText.getText()}`;
+        return jclToSubmit;
+    }
+
+    private generateLibs(libName: string, params: string[]) {
+        return params.map((str, i) => {
+            if (i === 0) {
+                return `//${libName}  DD  DISP=SHR,DSN=${str}`;
+            }
+            return `//         DD  DISP=SHR,DSN=${str}`;
+        }).join("\r\n");
+    }
+
+    private handleError(error: any) {
+        this.writeEmitter.fire(`encountered error ...\r\n`);
```

Comment:
Have you tested how this behaves on linux? 

If needed there is EOL constant that is platform specifice
```js
import { EOL } from 'os';
```

Reply:
Used this. But the line separator coming from the compile scripts are shown as \n and by EOL as \r\n